ddev/ddev-phpmyadmin
# fix: make `ddev phpmyadmin` work again for Gitpod, fixes #15

## The Issue

- #15

## How This PR Solves The Issue

Fixes the condition to open the correct port (I chose not to change the forwarded port from `8036` to `8037` in the `docker-compose.phpmyadmin-norouter.yaml` file because we only "use" HTTP ports in the cloud and it would be confusing if we forwarded the HTTPS port).

Adds a tip about preview mode in Gitpod.

Renames `docker-compose.phpmyadmin-norouter.yaml` to `docker-compose.phpmyadmin_norouter.yaml`

## Manual Testing Instructions

Test with Gitpod on DDEV HEAD:

[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/ddev/ddev/)

```
ddev get https://github.com/ddev/ddev-phpmyadmin/tarball/20240712_stasadev_launch_gitpod
ddev restart
ddev phpmyadmin
```

No need to test it with Codespaces because it won't work anyway, I see we don't omit `ddev-router` in Codespaces, it's in my TODOs.

## Automated Testing Overview



## Related Issue Link(s)

## Release/Deployment Notes
